### PR TITLE
Fix typos and tweak grammar.

### PR DESCRIPTION
**Pull request by jeannekamikaze**

### CODE REVIEW

**Comment by jeannekamikaze** [comment]
I have also noticed inconsistencies in the use of the term "open source". I've seen open source, open-source, and Open Source. Is there a guideline for this? As an example, the FSFE maintains a word list here, and e.g. free software is "Free Software":

https://fsfe.org/contribute/translators/wordlist.en.html

File: documentation/content/en/books/handbook/introduction/_index.adoc [hunk 1]
```diff
@@ -100,10 +100,10 @@ Here is just a sampling of some of the applications in which people are currentl
...
 * _Networking:_ Need a new router? A name server (DNS)? A firewall to keep people out of your internal network? FreeBSD can easily turn that unused PC sitting in the corner into an advanced router with sophisticated packet-filtering capabilities.
-* _Embedded:_ FreeBSD makes an excellent platform to build embedded systems upon. With support for the ARM(R), MIPS(R) and PowerPC(R) platforms, coupled with a robust network stack, cutting edge features and the permissive extref:{faq}[BSD license, bsd-license-restrictions] FreeBSD makes an excellent foundation for building embedded routers, firewalls, and other devices.
+* _Embedded:_ FreeBSD makes an excellent platform to build embedded systems upon. With support for the ARM(R), MIPS(R) and PowerPC(R) platforms, coupled with a robust network stack, cutting edge features and the permissive extref:{faq}[BSD license, bsd-license-restrictions], FreeBSD makes an excellent foundation for building embedded routers, firewalls, and other devices.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/introduction/_index.adoc:106]: While here, needs a serial comma after "cutting edge features".

File: documentation/content/en/books/handbook/introduction/_index.adoc [hunk 2]
```diff
@@ -118,19 +118,19 @@ FreeBSD has been known for its web serving capabilities - sites that run on Free
 FreeBSD's advanced features, proven security, predictable release cycle, and permissive license have led to its use as a platform for building many commercial and open source appliances, devices, and products.
 Many of the world's largest IT companies use FreeBSD:
 
-* link:http://www.apache.org/[Apache] - The Apache Software Foundation runs most of its public facing infrastructure, including possibly one of the largest SVN repositories in the world with over 1.4 million commits, on FreeBSD.
+* link:http://www.apache.org/[Apache] - The Apache Software Foundation runs most of its public facing infrastructure on FreeBSD, including possibly one of the largest SVN repositories in the world with over 1.4 million commits.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/introduction/_index.adoc:121]: While here, "public facing" should be "public-facing".

File: documentation/content/en/books/handbook/introduction/_index.adoc [hunk 3]
```diff
@@ -141,13 +141,13 @@ Many of the world's largest IT companies use FreeBSD:
...
-* link:https://www.midnightbsd.org[MidnightBSD] is a FreeBSD derived operating system developed with desktop users in mind. It includes all the software you'd expect for your daily tasks: mail, web browsing, word processing, gaming, and much more.
+* link:https://xigmanas.com/[XigmaNAS] - A file server distribution based on FreeBSD with a PHP-powered web interface.
+* link:http://www.opnsense.org/[OPNSense] is an open source, easy-to-use and easy-to-build FreeBSD-based firewall and routing platform. OPNsense includes most of the features available in expensive commercial firewalls, and more in many cases. It brings the rich feature set of commercial offerings with the benefits of open and verifiable sources.
+* link:https://www.midnightbsd.org[MidnightBSD] is a FreeBSD-derived operating system developed with desktop users in mind. It includes all the software you would expect for your daily tasks: mail, web browsing, word processing, gaming, and much more.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/introduction/_index.adoc:150]: While here, adjust for documentation style rule that says not to use "you". Something like "software expected for daily tasks".

**Review by pauamma — COMMENTED** [review]
Looks good, thanks! While reviewing, I noticed a few other nits in the lines this is touching, and it would make sense to fix all in one stroke.

**Comment by pauamma** [comment]
> I have also noticed inconsistencies in the use of the term "open source". I've seen open source, open-source, and Open Source. Is there a guideline for this? 

Not in our house style guide, https://docs.freebsd.org/en/books/fdp-primer/writing-style/. Perhaps there should be.

File: documentation/content/en/books/handbook/introduction/_index.adoc (hunk 1 above)
Comment (jeannekamikaze) [reply documentation/content/en/books/handbook/introduction/_index.adoc]: Done, thank you.

**Comment by jeannekamikaze** [comment]
Made changes based on feedback, thank you.

**Comment by jeannekamikaze** [comment]
I am not authorized to merge any of these changes, so please go ahead.

**Review by pauamma — APPROVED** [review]
LGTM now. Thanks.

**Comment by dbaio** [comment]
Hi.

Please, could you rebase this PR?

Thank you in advance.

And thanks for all your contributions.

**Comment by jeannekamikaze** [comment]
Done. Looks like somebody else edited one of the same lines.

**Comment by dbaio** [comment]
Committed on ff0bdebee5553703202c486ac63796140f6396bf. Thank you.

**Pull request closed without merging**